### PR TITLE
template enterprise versions doc

### PR DESCRIPTION
**Pull request by fredrikhgrelland**
Updating documentation based on addition of enterprise switches.

### CODE REVIEW

File: template/README.md [hunk 1]
```diff
@@ -20,36 +20,47 @@ The vagrant box ships with a default startup scheme. `make` from this directory
...
 Detailed documentation in [dev/vagrant/conf/README.md](dev/vagrant/conf/README.md)
 
-### Nomad ACLs
+### Pre packaged configuration switches
 
-| default   | environment variable  |  value  |
-|:---------:|:----------------------|:-------:|
-|           | nomad_acl             |  true   |
-| x         | nomad_acl             |  false  |
+The box comes standard with a set of environment switches to simplify testing of different scenarios and enable staged development efforts.
```
Comment (pdmthorsrud) [inline template/README.md:26]:
```suggestion

#### Enterprise vs Open Source Software (OSS)
As long as Enterprise is not set to `true` the box will utilise OSS version of the binaries. 
```

Comment (fredrikhgrelland) [reply template/README.md:26]: Did you mean to remove `The box comes standard with a set of environment switches to simplify testing of different scenarios and enable staged development efforts.` ?

Comment (fredrikhgrelland) [reply template/README.md:26]: See 5972815

Comment (pdmthorsrud) [reply template/README.md:26]: Sorry for late answer, but yeah, last commit is exactly what I tried to suggest!

**Pull request merged**